wolfssl/wolfcrypt/asn.h

```diff
@@ -76,11 +76,11 @@ that can be serialized and deserialized in a cross-platform way.
 #endif
 
 enum {
-    ISSUER  = 0,
-    SUBJECT = 1,
+    ASN_ISSUER  = 0,
```

Comment:
Customers will be using ISSUER, SUBJECT and so on.
How can they still work?
My preference is to put a #ifdef protection around the defines from old to new.

Reply:
fixed with remapping macros that are enabled by default.  each macro has an additional gate to inhibit it if it would attempt to redefine -- this way, applications with colliding macros may be able to avoid conflicts just by including headers in the right order.